templates/taxi/car_detail.html

```diff
@@ -1,12 +1,19 @@
 {% extends "base.html" %}
-
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>
+    <p>Manufacturer: {{ car.manufacturer.name }} </p>
+    <p>Model: {{ car.model }} </p>
+    <p> country: {{ car.manufacturer.country }}</p>
+  </h1>
+  <a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary">Update</a>
+  <a href="{% url 'taxi:car-delete' car.id %}" class="btn btn-danger">Delete</a>
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.